src/tests/unit/gna/gna_cpp_wrapper_test.cpp

```diff
@@ -1,4 +1,4 @@
-// Copyright (C) 2018-2022 Intel Corporation
+// Copyright (C) 2022 Intel Corporation
```

Comment:
Since file is old keep
```suggestion
// Copyright (C) 2018-2022 Intel Corporation
```